src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/S3Configuration.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.s3.S3Client
+import java.net.URI
+
+@Configuration
+class S3Configuration(
+  @Value("\${aws.s3.provider}") private val provider: String,
+  @Value("\${aws.s3.region}") private val region: String,
+  @Value("\${aws.s3.access-key-id}") private val accessKeyId: String,
+  @Value("\${aws.s3.secret-access-key}") private val secretAccessKey: String,
+  @Value("\${aws.s3.endpoint.uri:}") private val uri: String,
```

Comment:
Still need to remove ':' here.
Sorry I commented on the wrong PR for this specific case.

Reply:
see the comment in the other PR - this is intentional.